tests/config/config.d/storage_conf.xml

```diff
@@ -19,7 +19,7 @@
                 <type>cache</type>
                 <disk>s3_disk</disk>
                 <path>s3_cache/</path>
-                <max_size>104857600</max_size>
+                <max_size>209715200</max_size>
```

Comment:
Test was failing because we expect in the test to successfully reserve space but there was `No space left in cache to reserve 46 bytes, reason: cannot evict enough space, will continue without cache download`. Near that time cache state was `Current usage 104256706/104857600 in size, 94/10000000 in elements count`, and ` Collected 2 eviction candidates, skipped 92 candidates while iterating` means that those 92 were unreleasable because of concurrent queries or merges (table `s3_hits` is used for this test, this table is used in several tests + the cache object used for this table is also used for other tests)